src/styles/base.scss

```diff
@@ -1 +1,37 @@
 /*Add global styles here*/
+
+@import './variables.scss';
+@import './mixins.scss';
+
+/*Font sizes*/
+.fs-12{
```

Comment:
@aka-jain Please fix indentation in this css file. 2 spaces indentation will be perfect. also put 1 space before `{`